.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: python
+cache: pip
+
+python:
+  - "3.7"
+
+install:
+  - pip install -r requirements-test.txt
+  - pip install coveralls
```

Comment:
requirements-travis.txt ?

```
-r ./requirements-test.txt

coveralls

Reply:
Good suggestion. However, since travis has its own config, I find it OK to just include that dependency there. It shouldn't grow a lot the list of dependencies for travis, so I rather don't add a new file for that